libs/core/src/lib/split-button/split-button.component.ts

```diff
@@ -67,6 +77,24 @@ export class SplitButtonComponent {
     @Input()
     expandButtonAriaLabel = 'More';
 
+    /** Selected menu item */
+    @Input()
+    selected: MenuItemComponent;
+
+    /** Whether or not the element should keep a fixed width. The width could change if the text changes length. */
+    @Input()
+    fixedWidth = true;
+
+    /** Whether or not the main action title should remain set as the button's action after another option is selected. */
+    @Input()
+    keepMainAction = false;
+
+    /**
+     * The object that contains the mainActionTitle and the callback function that should be executed when the button is clicked.
+     */
+    @Input()
```

Comment:
Since itll take a callback we can also remove the primaryButtonClicked output

Reply:
I initially thought the same, but figured it is possible there could be cases when the dev needs to know if the primary button is clicked even though it's been changed to a different option.  Doesn't hurt to leave it, I think